drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/base.rb

```diff
@@ -15,6 +15,7 @@ class Base < ::HudReports::QuestionBase
     include HudReports::Veterans
     include HudReports::LengthOfStays
     include HudReports::Incomes
+    include HudApr::CellDetailsConcern
```

Comment:
Is this providing any functionality to Generator?
I'm thinking maybe the right way to do this, since we didn't end up delegating all of the `column_headings` complexity to the questions, is to do what you had planned, and have it take a "question number" then include it here instead of in QuestionBase, and pass in `@question` in the view so you end up with `@generator.column_headings(@question)` (I think all the other functionality can pretty much remain the same)

Reply:
✅ Performance is also significantly better if we aren't instantiating a question